utils/docker/Dockerfile.centos.7

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright 2018-2019, Intel Corporation
+#
+# 'recipe' for Docker to build an image of centOS-based
+# environment for building the DAOS project.
+#
+
+# Pull base image
+FROM centos:7
+MAINTAINER Johann Lombardi <johann.lombardi@intel.com>
+
+# Build arguments can be set via -build-arg
+# use same UID as host and default value of 1000 if not specified
+ARG UID=1000
+
+# Update distribution
+#Nothing to do for CentOS
+
+# Install basic tools
+RUN yum install -y epel-release; \
+    yum install -y \
```

Comment:
(For all occurrences, not just this one) should move the `-y` argument to before `install` since it's a `yum` command switch not an `install` switch.

This is how the cart Dockerfile file is and having them as similar as possible makes it easier to notice see differences between them.

Reply:
Fixed in next update